src/IW/Sync/Update.hs

```diff
@@ -30,13 +42,22 @@ syncRepos
        )
     => Integer -- ^ The starting date interval used in the search function
     -> m ()
-syncRepos interval = do
+syncWithGithub interval = do
+    log I "Starting synchronization of all repositories and issues..."
     today <- liftIO getToday
+    log I "Searching for all Haskell repositories..."
     repos <- searchAllHaskellRepos today interval
+    log I "Upserting repositories into the database..."
```

Comment:
Logging is very useful in this function! And we definitely should have it. It's just unfortunate that because of calls to logging functions the code becomes less readable and all steps are not apparent immediately... It's an open question, how to improve the situation though. But at least, these logging messages serve as a good documentation

Reply:
@chshersh Yes, I agree. I thought the same, but I think it’s definitely worth it. This was my first time using any logging library but I really enjoy how easy and straightforward it is to use. Very user friendly.